epix/io.py

```diff
@@ -151,18 +152,7 @@ def loader(directory, file_name, arg_debug=False, outer_cylinder=None, kwargs_up
 # ----------------------
 # Outputing wfsim instructions:
 # ----------------------
-int_dtype = [(('Waveform simulator event number.', 'event_number'), np.int32),
-             (('Quanta type (S1 photons or S2 electrons)', 'type'), np.int8),
-             (('Time of the interaction [ns]', 'time'), np.int64),
-             (('X position of the cluster[cm]', 'x'), np.float32),
-             (('Y position of the cluster[cm]', 'y'), np.float32),
-             (('Z position of the cluster[cm]', 'z'), np.float32),
-             (('Number of quanta', 'amp'), np.int32),
-             (('Recoil type of interaction.', 'recoil'), np.int8),
-             (('Energy deposit of interaction', 'e_dep'), np.float32),
-             (('Eventid like in geant4 output rootfile', 'g4id'), np.int32),
-             (('Volume id giving the detector subvolume', 'vol_id'), np.int32)
-             ]
+int_dtype = wfsim.instruction_dtype
```

Comment:
I think I would make it the other way around and keep the `dtype` here and get it from epix in WFsim. Depending on how we will make effective solutions we would maybe need to change the `dtype` here but not in WFsim.

Reply:
This goes against everything I've been saying about WFsim/fast-sim code duplication, but I would just keep the definition in both, without crossed dependencies; simply to improve code readability and to allow for this to be independent in the future, even if for the moment this isn't the case.

Reply:
Epix is one of the possible ways to provide instructions to wfsim. Since there are other it makes no sense to put the definition here